control/tests/sisotool_test.py

```diff
@@ -88,7 +119,21 @@ def test_sisotool(self, sys):
         step_response_moved = np.array(
             [0., 0.0072, 0.0516, 0.1554, 0.3281, 0.5681, 0.8646, 1.1987,
              1.5452, 1.875])
-        # old: array([0., 0.0239, 0.161 , 0.4547, 0.8903, 1.407,
-        #             1.9121, 2.2989, 2.4686, 2.353])
         assert_array_almost_equal(
             ax_step.lines[0].get_data()[1][:10], step_response_moved, 4)
+
+        # test supply tvect
+        sisotool(sys, tvect=np.arange(0, 1, .1))
```

Comment:
Somehow this still does not cover https://coveralls.io/builds/36911686/source?filename=control%2Fsisotool.py#L168

Reply:
Let's see if that works